packages/components/radio-group/src/utils/helper-functions.js

```diff
@@ -0,0 +1,14 @@
+import { keyboardKeys } from './constants';
+
+export default function getDirectionByKey(keyCode) {
+	switch (keyCode) {
+		case keyboardKeys.ARROW_RIGHT:
+		case keyboardKeys.ARROW_DOWN:
+			return 1;
```

Comment:
These codes are not easy to understand, can you please change them to some constants that shows what they mean?

Reply:
I suggest having DIRECTION_FORWARD and DIRECTION_BACKWARD constants. Or just FORWARD and BACKWARD. Could the default zero value be called DIRECTION_NONE?

Reply:
Done